docs/source/WritingTests.rst

```diff
@@ -208,6 +208,10 @@ to be accessed from up to three sources:
    ``MyTest.test_foo``, with variant ``debug-ffff``, the data directory path
    will be ``/home/user/test.py.data/MyTest.test_foo/debug-ffff/``.
 
+.. note:: Unlike INSTRUMENTED tests the ``SIMPLE`` tests only define `file`
+          and `variant` data_dirs, therefor the most-specific data-dir
```

Comment:
Final note here is that I think "therefore" is the word you were looking. Therefor is a valid word, but AFAICT, has a slight different meaning: https://www.grammarly.com/blog/therefore-vs-therefor/

Reply:
yep, sorry, it's because I learned English from movies and I suck at written form...